js/components/cohort-definition-browser.js

```diff
@@ -94,14 +79,21 @@ define([
 					title: 'Author',
 					className: 'author-column',
 					render: datatableUtils.getCreatedByFormatter(),
-				}
+				},
 			];
 
-			this.rowClick = this.rowClick.bind(this);
-		}
-
-		rowClick(data) {
-			this.action(() => this.selected(data));
+			if (!!this.multiChoice) {
+				this.columns = [
```

Comment:
Why wouldn't you move this into the EntityBrowser since it is a common wrapper which takes `multiChoice` parameter? This way the code block would appear just once and wouldn't be repeated across "client" components

Reply:
fixed